src/Network/Network/help/Remove-AzFirewallPolicy.md

```diff
@@ -0,0 +1,201 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/remove-azfirewallpolicy
+schema: 2.0.0
+---
+
+# Remove-AzFirewallPolicy
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
+
+## SYNTAX
+
+```
+Remove-AzFirewallPolicy -Name <String> -ResourceGroupName <String> [-Force] [-PassThru] [-AsJob]
+ [-ResourceId <String>] -InputObject <PSAzureFirewallPolicy> [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
```

Comment:
From you code I see 3 parameter sets, but only one here. I suppose you need to regenerate readme

Reply:
You are right. Thank you for pointing this out. Regenerated the file.